app/javascript/src/components/Dashboard/Contacts/index.jsx

```diff
@@ -16,14 +18,17 @@ const Contacts = () => {
   const [contacts, setContacts] = useState(TABLE_ROW_DATA);
   const [showDeleteAlert, setShowDeleteAlert] = useState(false);
   const [selectedContactId, setSelectedContactId] = useState(null);
+  const [showNewContactPane, setShowNewContactPane] = useState(false);
+  const [showEditContact, setShowEditContact] = useState(false);
+  const [selectedContact, setSelectedContact] = useState({});
 
   const { t } = useTranslation();
 
   const deleteContact = () => {
-    const nonDeletedNotes = contacts.filter(
+    const nonDeletedContacts = contacts.filter(
       contact => contact.id !== selectedContactId
     );
-    setContacts(nonDeletedNotes);
+    setContacts(nonDeletedContacts);
```

Comment:
What's your strong reason for not using the previous state of the setter?

Reply:
I didn't think about filtering the contacts inside the setter function. I'll update it and follow that from now on